apps/sensenet/cypress/integration/ui/logonavigation.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Logo Navigation Test', () => {
+  beforeEach(() => {
+    cy.login('admin')
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+
+  it('should go to Root on click of Logo', () => {
+    cy.get('a[href="/content/explorer/"]').click()
+    cy.get('[alt="logo"]')
```

Comment:
It is a too general selector, can easily select unwanted dom elements too. Feel free to put custom data attribute to the element for testing purposes, eg. data-test="sensenet-logo"
(logo's code is in desktop-app-bar.tsx file)

Reply:
changed in latest commit, but can't test if test is working now since update to the desktop-app-bar.tsx is not pushed to the remote repository offcourse..